<launch>
    
    <node pkg="wpirm_navigation" name="cone_vision_pose" type="cone_vision_pose.py" output="screen">
        <param name="camera_frame" type="str" value="" />
        <param name="detection_topic" type="str" value="/yolov2_ros/detection" />
        <param name="odom_topic" type="str" value="/zed/odom" />
        <param name="horiz_fov" type="double" value="96.0" />
    </node>
</launch>